extensions/api/data-management/transferprocess/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/transferprocess/model/TransferProcessDto.java

```diff
@@ -18,13 +18,16 @@
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
 
+import java.util.Map;
+
 @JsonDeserialize(builder = TransferProcessDto.Builder.class)
 public class TransferProcessDto {
     private String id;
     private String type;
     private String state;
     private String errorDetail;
     private DataRequestDto dataRequest;
+    private Map<String, String> dataDestination = Map.of();
```

Comment:
For coherence with the `asset` data management api, a dedicated `DataAddressDto` object could be used instead

Reply:
@ndr-brt would that object only contain a map? Since the values (keys) are dependent on the address type, not sure what that would bring?

Reply:
Yes, it does not bring much, it's just for coherence, if here the `DataAddress` is represented as a `Map`, the same should be on the `asset` api.

Reply:
It turns out the `DataAddressDto` already exists :) will reuse

Reply:
Added `DataAddressInformationDto` (to avoid name conflict with `DataAddressDto` used in `asset`)

Reply:
@algattik I don't think they have to be called differently, as they are in different modules and have different packages, also `ContractAgreementDto` has two different definitions